packages/edge-gateway/test/index.spec.js

```diff
@@ -15,4 +15,54 @@ test('Gets content from binding', async (t) => {
   )
   await response.waitUntil()
   t.is(await response.text(), 'Hello nftstorage.link! 😎')
+
+  // CSP
+  const csp = response.headers.get('content-security-policy') || ''
+  t.true(csp.includes("default-src 'self' 'unsafe-inline' 'unsafe-eval'"))
+  t.true(csp.includes('blob: data'))
+  t.true(csp.includes("form-action 'self' ; navigate-to 'self';"))
+})
+
+test('Gets content with no csp header when goodbits csp bypass tag exists', async (t) => {
```

Comment:
we should take a moment to see if there is a much more permissive CSP header we can provide rather than dropping it completely...

Reply:
I guess we will think about that as other use cases rise in the goodbits and with tags. Until we spec a better solution, let's just keep the basics so that users with broken CIDs are not affected :)